### DISCUSSION TITLE
Nice

### DISCUSSION THREAD

**Discussion by wolfgang-ch** [mytourbook/mytourbook#912] [General]
![image](https://user-images.githubusercontent.com/1283445/184524451-c3e6fed8-275e-47b2-9776-22d93adf57fa.png)

![image](https://user-images.githubusercontent.com/1283445/184524433-3ae098fb-8bb8-450e-85ce-bed99ea49cfd.png)

**Comment by telemaxx** [comment]
![Screenshot_20220814-080624_Samsung Internet](https://user-images.githubusercontent.com/22706487/184524723-a5c4c063-0ba4-406e-ad68-1b4605eac286.jpg)

**Reply by telemaxx** [reply]
nice Animation

**Comment by FJBDev** [comment]
Pretty cool

![image](https://user-images.githubusercontent.com/25235665/184554990-9de9d269-9f09-4f7a-82dc-fcce9a174c00.png)


It makes me think of [Gource](https://gource.io/), an animation representing the commits, branches, releases...etc

https://www.youtube.com/watch?v=NjUuAuBcoqs&t=75s
